addons/addon-environment-sc-ui/packages/environment-type-mgmt-ui/src/parts/environment-types/EnvTypeCard.js

```diff
@@ -94,7 +95,7 @@ class EnvTypeCard extends Component {
     );
 
     return (
-      <Card key={`et-${envType.id}`} raised className="mb3">
+      <Card data-testid="envtypecard" key={`et-${envType.id}`} raised className="mb3">
```

Comment:
Sorry for not seeing this in the original PR. For the convention of the `data-testid`, let's keep it consistent as `kebab-case`. That's the format of the other `ids`.
`env-type-card`

Reply:
will do!